client/signup/steps/domains/index.jsx

```diff
@@ -681,6 +681,7 @@ class DomainsStep extends React.Component {
 		}
 
 		const { flowName, isAllDomains, translate, sites } = this.props;
+		const source = get( this.props, 'queryObject.source' );
```

Comment:
Do we actually need `lodash` for this? 

```suggestion
		const source = get( this.props, 'queryObject.source' );
		const {source} = this.props.queryObject;
```

Reply:
`lodash` is used throughout the file, so for the sake of consistency it makes sense to carry on using it here.

Also if `queryObject` was `undefined` your suggested change would throw an error. `lodash` provides safety checks around this.

Reply:
> so for the sake of consistency it makes sense to carry on using it here.

> `lodash` provides safety checks around this.

Yes, that's true. When `queryObject` is undefined my suggestion would throw an error and `lodash` covers this. My concern is that `lodash` is quite an elaborate library and adds weight to the codebase. So we'd not overuse it but, for the reasons mentioned above, it makes sense to keep using it!